README.md

```diff
@@ -129,6 +129,7 @@ Sniffs the DHT gossip and indexes file and directory hashes. Metadata and search
 
 ## Tools
 
+- [@agentofuser/ipfs-deploy](https://github.com/agentofuser/ipfs-deploy) - Zero-config CLI to deploy static websites: cd my-static-website && npx @agentofuser/ipfs-deploy
```

Comment:
Should we call it `Ipfs-deploy` ?

Reply:
@koalalorenzo I changed it to `ipfs-deploy` only. Thank you!